Library/ViewModels/AddNewCardViewModel.swift

```diff
@@ -36,6 +37,7 @@ public protocol AddNewCardViewModelOutputs {
   var dismissKeyboard: Signal<Void, Never> { get }
   var paymentDetails: Signal<PaymentDetails, Never> { get }
   var paymentDetailsBecomeFirstResponder: Signal<Void, Never> { get }
+  var reusableCardSwitchIsHidden: Signal<Bool, Never> { get }
```

Comment:
Does this name not match `reusableCardStackViewContainer` intentionally for it being too long?